tmt/schemas/common.yaml

```
@@ -142,6 +166,56 @@ definitions:
         - type: boolean
         - type: number
 
+  # https://tmt.readthedocs.io/en/stable/spec/plans.html#context
+  # https://tmt.readthedocs.io/en/stable/spec/context.html#spec-context
+  fmf_context:
```

```suggestion
  context:
```

Let's rename this one as well to make it consistent. The keys `distro, component, trigger...` and friends defined here are not an `fmf` thing. They are part of the `tmt` spec.

---

Renamed in 418a0507